content/docs/en/elements/components/list-picker.md

```diff
@@ -1,5 +1,25 @@
 ---
 title: ListPicker
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_list_picker_.listpicker
 ---
 
-TODO
+The Listpicker component allows you to pick a value from a list.
+
+---
+
+```html
+<ListPicker :items="pokemons" selectedIndex="0" @selectedIndexChange="selectedIndexChanged(event)"></ListPicker>
+```
+[> screenshots for=ListPicker <]
+
+## Props
+
+| name | type | description |
+|------|------|-------------|
+| `items` | `Array<string>` | An Array of strings that will be options of the list
```

Comment:
I think this should be changed to something like this:
```diff
- An Array of strings that will be options of the list
+ An Array of strings that will be displayed as options in the list picker.
```

Reply:
Makes more sense, yeah